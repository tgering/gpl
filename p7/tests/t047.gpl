
double x = 3.14159;
double y = 3.14159;

on akey
{
    x -= .005;
    y -= 1;
}

on skey
{
    print("x = " + x);
    print("y = " + y);
}
