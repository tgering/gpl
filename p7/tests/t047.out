gpl.cpp::main()
  input file(tests/t047.gpl)
  random seed(42)
  read_keypresses_from_standard_input(true)
  dump_pixels(false)
  symbol_table(true)
  graphics(true)

gpl.cpp::main() Calling yyparse()


gpl.cpp::main() after call to yyparse().


Printing the symbol table from main()
double x = 3.14159
double y = 3.14159
gpl.cpp::main() Calling window->initialize().


gpl.cpp::main() Passing control to window->main_loop().
print[13]: x = 3.13659
print[14]: y = 2.14159

User quit program. Printing the symbol table.
double x = 3.13659
double y = 2.14159
